Assets/_BForBoss/_Core/Scripts/Audio/AudioSettingsModel.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections;
+using System.Collections.Generic;
+using Perigon.Utility;
+using UnityEngine;
+
+namespace BForBoss
```

Comment:
Do you think it would be beneficial to have a `BForBoss.Audio` namespace

Reply:
I could certainly namespace some of the more internal classes, I don't know that it makes sense to move it into its own assembly or anything like that though